### DISCUSSION TITLE
Gantt Chart

### DISCUSSION THREAD

**Discussion by aalex675** [Live-Charts/LiveCharts2#538] [Q&A]
In v0, it was possible to make a Gantt chart by creating a RowSeries of GanttPoint, but I can't figure out how to do that in v2. I've tried creating my own custom type with a map and setting various combinations of Values (Primary/Secondary/Tertiary/etc), but can't seem to get the start of a bar to be offset from 0. Should this be possible? Is this not implemented yet or has this feature been dropped?

**Comment by dev11ocean** [comment]
> In v0, it was possible to make a Gantt chart by creating a RowSeries of GanttPoint, but I can't figure out how to do that in v2. I've tried creating my own custom type with a map and setting various combinations of Values (Primary/Secondary/Tertiary/etc), but can't seem to get the start of a bar to be offset from 0. Should this be possible? Is this not implemented yet or has this feature been dropped?

Any luck on this? I've been trying the same.

**Comment by dbenarfa** [comment]
I hope you have come to a solution to create a Gantt chart with LiveCharts2

**Comment by michallabuzsybilla** [comment]
Need it as well! One solution is to do it just using Sections but its more a hack than solution

**Comment by planetarian** [comment]
still looking for info on how to do this in 2024..

**Comment by dev11ocean** [comment]
Any updates on this?
